app/assets/stylesheets/application.scss

```diff
@@ -170,12 +174,20 @@ ul {
 }
 
 .member-list {
+  background-color: rgba(244, 162, 23, 0.66);
+  -moz-border-radius: 15px;
+  -webkit-border-radius: 15px;
+  border-radius: 15px;
+  margin: 20px;
```

Comment:
Properties should be ordered background-color, -moz-border-radius, -webkit-border-radius, border-radius, color, list-style-type, margin, margin-left, margin-right, max-height